src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -839,9 +839,30 @@ void ApplicationManagerImpl::ConnectToDevice(const std::string& device_mac) {
 
 void ApplicationManagerImpl::OnHMIStartedCooperation() {
```

Comment:
Am I right that this function will be called On BC.IsReady? 
Confusing that after calling `OnHMIStartedCooperation` hmi_cooperating_ may be still false

Reply:
[done](https://github.com/smartdevicelink/sdl_core/pull/3269/files#diff-6289bfe1988a528c2f109887d563c017R840)